src/keria/app/delegating.py

```diff
@@ -172,4 +178,71 @@ def processPartialWitnessEscrow(self):
 
                 self.hby.db.dpwe.rem(keys=(pre, said))
                 self.hby.db.dune.pin(keys=(srdr.pre, srdr.said), val=srdr)
+                
+class DelegatorEnd:
+    """ Resource class for for handling delegator events"""
+    
+    def __init__(self, identifierResource) -> None:
+        """
+
+        Parameters:
+            identifierResource (IdentifierResourceEnd): endpoint class for creating rotation and interaction events
 
+        """
+        self.identifierResource = identifierResource
+
+    def on_post(self, req, rep, name):
+        """ Identifier delegator enpoint POST to create the ixn anchor and approve the delegation
+
+        Parameters:
+            req (Request): falcon.Request HTTP request object
+            rep (Response): falcon.Response HTTP response object
+            name (str): human readable name for Hab to rename
+
+        """
+        if not name:
+            raise falcon.HTTPBadRequest(description="name is required")
+        agent = req.context.agent
+        hab = agent.hby.habByName(name)
+
+        if hab is None:
+            raise falcon.HTTPNotFound(title=f"No AID with name {name} found")
+        
+        body = req.get_media()
+        
+        op = self.identifierResource.interact(agent, name, body)
+        anc = httping.getRequiredParam(body, "ixn")
+        
+        # successful approval returns the delegatee prefix
+        teepre = self.approveDelegation(agent, hab, anc)
+        adop = agent.monitor.submit(hab.kever.prefixer.qb64, longrunning.OpTypes.delegation,
+                                    metadata=dict(teepre=teepre, depends=op))
+        
+        try:
+            rep.status = falcon.HTTP_200
+            rep.content_type = "application/json"
+            rep.data = adop.to_json().encode("utf-8")
+            return rep
+        except (kering.AuthError, ValueError) as e:
+            raise falcon.HTTPBadRequest(description=e.args[0])
+    
+    @staticmethod
+    def approveDelegation(agent, hab, anc) -> str:
+        serder = serdering.SerderKERI(sad=anc)
+        
+        teepre = anc['a'][0]['i']
+        teesaid = anc['a'][0]['d']
+
+        for (pre, sn), dig in hab.db.delegables.getItemIter():
+            if pre == teepre:
+                seqner = coring.Seqner(sn=serder.sn)
+                couple = seqner.qb64b + serder.saidb
+                dgkey = dbing.dgKey(coring.Saider(qb64=teepre).qb64b, coring.Saider(qb64=teesaid).qb64b)
+                # the dip event should have been received from the delegatee via a postman call
+                # and will be sitting in the delegator escrows (hence the hab.db.delegables above)
+                # adding the authorize event seal will allow the dip to be processed
+                # and added to the delegator kever
+                hab.db.setAes(dgkey, couple)  # authorizer event seal (delegator/issuer)
+                return teepre
```

Comment:
I'm still not seeing where you reprocess the delegate's event from delegables and then delete it from delegables.

Reply:
Thank you @pfeairheller ! based on my debugging, I see the order of operations as:
- [after the delegatee completes witness receipts of the dip, it is sent to the delegator](https://github.com/WebOfTrust/keripy/blob/38cc2891286afce6c7426b0bd14abca287f229fc/src/keri/app/delegating.py#L186)
- [this dip event is escrowed by the delegator](https://github.com/WebOfTrust/keripy/blob/38cc2891286afce6c7426b0bd14abca287f229fc/src/keri/core/eventing.py#L2949)
- The delegator submits the anchor ixn event, but that alone doesn't allow the escrowed dip to be processed. That is because the processEscrowDelegables continues to [fail this check](https://github.com/WebOfTrust/keripy/blob/38cc2891286afce6c7426b0bd14abca287f229fc/src/keri/core/eventing.py#L5800)
- We added an additional step of approving delegation which only calls setAes with the ixn anchor info.
- The next time the processEscrowDelegables runs, the Aes check passes which [allows the delegator to process the dip successfully](https://github.com/WebOfTrust/keripy/blob/38cc2891286afce6c7426b0bd14abca287f229fc/src/keri/core/eventing.py#L5806)
- And then for the [delegables to be removed](https://github.com/WebOfTrust/keripy/blob/38cc2891286afce6c7426b0bd14abca287f229fc/src/keri/core/eventing.py#L5830)
- This results in the delegator having the delegatee in its kevers and for the delegation to be complete

Reply:
Now I see the disconnect... processEscrowDelegables is never called anywhere.  Here or KERIpy.